packages/components/nodes/llms/Azure OpenAI/AzureOpenAI.ts

```diff
@@ -0,0 +1,129 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { AzureOpenAIInput, OpenAI, OpenAIInput } from 'langchain/llms/openai'
+
+class AzureOpenAI_LLMs implements INode {
+    label: string
+    name: string
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Azure OpenAI'
+        this.name = 'azureOpenAI'
+        this.type = 'AzureOpenAI'
+        this.icon = 'Azure.svg'
+        this.category = 'LLMs'
+        this.description = 'Wrapper around Azure OpenAI large language models'
+        this.baseClasses = [this.type, ...getBaseClasses(OpenAI)]
+        this.inputs = [
+            {
+                label: 'Azure OpenAI Api Key',
+                name: 'azureOpenAIApiKey',
+                type: 'password'
+            },
+            {
+                label: 'Model Name',
+                name: 'modelName',
```

Comment:
Refer to here for models: https://learn.microsoft.com/en-GB/azure/cognitive-services/openai/concepts/models